dotnet/src/Extensions/PromptTemplate.Handlebars/HandlebarsPromptTemplate.cs

```diff
@@ -1,60 +1,72 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
-using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-using HandlebarsDotNet;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.PromptTemplate.Handlebars.Helpers;
 
 namespace Microsoft.SemanticKernel.PromptTemplate.Handlebars;
 
-internal sealed class HandlebarsPromptTemplate : IPromptTemplate
+/// <summary>
+/// Represents a Handlebars prompt template.
+/// </summary>
+public class HandlebarsPromptTemplate : IPromptTemplate
```

Comment:
This needs to be public for the 1.0 release?

Reply:
Changed to internal